src/services/API/Event.js

```diff
@@ -10,4 +14,12 @@ export default {
         return response.data;
       });
   },
+  /**
+   *
```

Comment:
description for the method?

Reply:
As discussed, skippping it as method name is obvious